src/main/kotlin/org/malachite/estella/commons/models/offers/RecruitmentProcess.kt

```diff
@@ -17,10 +17,10 @@ data class RecruitmentProcess(
                 name = "recruitment_process_quizes",
                 joinColumns = [JoinColumn(name = "recruitment_processes_id")],
                 inverseJoinColumns = [JoinColumn(name = "quizes_id")]
-        ) val quizzes: Set<Quiz>,
+        ) val quizzes: Set<Quiz>?,
```

Comment:
Jak nie ma nic to po prostu jest pusty zbiór, nie trzeba tego robić nullowalnego